relay-server/src/metrics/outcomes.rs

```diff
@@ -67,28 +62,16 @@ impl MetricOutcomes {
                 }
             }
         }
-
-        // When rejecting metrics, we need to make sure that the number of merges is correctly handled
-        // for buckets views, since if we have a bucket which has 5 merges, and it's split into 2
-        // bucket views, we will emit the volume of the rejection as 5 + 5 merges since we still read
-        // the underlying metadata for each view, and it points to the same bucket reference.
-        // Possible solutions to this problem include emitting the merges only if the bucket view is
-        // the first of view or distributing uniformly the metadata between split views.
-        for bucket in buckets {
-            relay_log::trace!("{:<50} -> {outcome}", bucket.name());
-            self.metric_stats.track_metric(scoping, bucket, &outcome)
-        }
     }
 
     /// Tracks the cardinality of a metric.
     #[cfg(feature = "processing")]
     pub fn cardinality(
```

Comment:
Why does this function still need to exist?

Reply:
It doesn't currently, but I wanted to keep it isolated to metric stats. This is technically still useful for "Metric Outcomes".